components/main/Desktop/CouncilSelector/CouncilsList.jsx

```diff
@@ -44,9 +44,8 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
-function CouncilsList({ items, onClick }) {
+function CouncilsList({ items, onClick, searchTerm, setSearchTerm }) {
```

Comment:
On line 78 below in CouncilsList.propTypes, don't forget to add the propTypes for `searchTerm` and `setSearchTerm`.

```
searchTerm: PropTypes.string.isRequired;
setSearchTerm: PropTypes.func.isRequired;
```

It helps us maintain type safety and clarity about the expected types of props the component relies on, making it easier for anyone to understand the prop requirement at a glance. Most importantly it gets rid of the annoying eslint red warnings ;)